broadcast/src/buffered/ingress.rs

```diff
@@ -48,7 +48,7 @@ impl<D: Digest, M: Digestible<D>> Mailbox<D, M> {
     }
 }
 
-impl<D: Digest, M: Codec + Digestible<D>> Broadcaster for Mailbox<D, M> {
+impl<Cfg, D: Digest, M: Codec<Cfg> + Digestible<D>> Broadcaster<Cfg> for Mailbox<D, M> {
```

Comment:
In some places we enforce that `C`/`Cfg` is `Copy + Send + 'static` and in others we don't. I recommend defining a `Cfg` trait in `Codec` that implements `Copy + Send + 'static` with no required functions to force standardization here.